README.md

```diff
@@ -1,8 +1,8 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Hello World
 
->   Hello Microverse
+- html & css linters configuration
 
 
```

Comment:
[OPTIONAL] - Please don't forget to summarize the project while working on the README file.

example 

```
> Project created to display a welcome message and how to set up linters in a project.
```